### PR TITLE
Pytest Style analysis/test_nuclinfo.py

### PR DESCRIPTION
**Pull request by utkbansal**
Fixes #

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/analysis/test_nuclinfo.py [hunk 1]
```diff
@@ -35,134 +35,134 @@
...
 
 
-class TestNuclinfo(TestCase):
-    def setUp(self):
-        self.u = mda.Universe(NUCL)
+class TestNuclinfo(object):
+    
+    @staticmethod
+    @pytest.fixture()
+    def u():
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/analysis/test_nuclinfo.py]: can we increase the scope of this fixture?

Comment (utkbansal) [inline testsuite/MDAnalysisTests/analysis/test_nuclinfo.py]: @richardjgowers Increase to what level?

**Comment by utkbansal** [comment]
@richardjgowers Done. Dropped the class too.

File: testsuite/MDAnalysisTests/analysis/test_nuclinfo.py [hunk 2]
```diff
@@ -35,134 +34,162 @@
...
+    assert_almost_equal(val, 4.601, decimal=3)
 
-    def test_minor_pair_1(self):
-        val = nuclinfo.minor_pair(self.u, 3, 17, seg1='RNAA', seg2='RNAA')
-        assert_almost_equal(val, 16.649, decimal=3)
 
-    def test_minor_pair_2(self):
-        val = nuclinfo.minor_pair(self.u, 20, 5, seg1='RNAA', seg2='RNAA')
-        assert_almost_equal(val, 4.356, decimal=3)
+def test_minor_pair_1(u):
```
Comment (jbarnoud) [inline testsuite/MDAnalysisTests/analysis/test_nuclinfo.py]: `test_minor_pair_1` and `test_minor_pair_2` can be grouped in a parametrized `test_minor_pair`.

File: testsuite/MDAnalysisTests/analysis/test_nuclinfo.py [hunk 3]
```diff
@@ -35,134 +34,162 @@
...
+    assert_almost_equal(val, 4.356, decimal=3)
 
-    def test_phase_cp_1(self):
-        val = nuclinfo.phase_cp(self.u, seg='RNAA', i=9)
-        assert_almost_equal(val, 14.846, decimal=3)
 
-    def test_phase_cp_2(self):
-        val = nuclinfo.phase_cp(self.u, seg='RNAA', i=21)
-        assert_almost_equal(val, 16.199, decimal=3)
+def test_major_pair_1(u):
```
Comment (jbarnoud) [inline testsuite/MDAnalysisTests/analysis/test_nuclinfo.py]: `test_major_pair_*` can be grouped in a parametrized test.

**Pull request merged**